src/components/MapFilter/MapFilter.js

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import { NavLink, Image } from '@mantine/core';
+import React, { useState, useCallback } from 'react';
+import MapFilterRadio from './MapFilterRadio';
+import MapFilterSwitch from './MapFilterSwitch';
+
+const MapFilter = () => {
+  const [filterValues, setFilterValues] = useState({
+    disasterArea: 'วันนี้',
+    disasterFactors: {},
+    populationData: {}
+  });
+
+  const mockData = {
```

Comment:
Not sure if we will get these from BE or need to hardcode this on our side. 
If we going to hardcode, maybe I will move it to another place.

Reply:
For now hard code is ok krub, but might need to put it in constant file